app/views/foreman/unattended/finish-katello.erb

```diff
@@ -60,6 +60,8 @@ cat > /etc/puppet/puppet.conf << EOF
 <%= snippet 'puppet.conf' %>
 EOF
 
+<%= snippet('remote_execution_ssh_keys') %>
```

Comment:
Incidentally, I was thinking more that maybe we should move the Katello's templates to community-templates, it'd be more likely to be kept up-to-date.  Or abandoning it all together, and relying on Foreman's default kickstart. I forget why we decided not to last time we talked about it.